utils/ctranslate.py

```diff
@@ -56,5 +65,8 @@ def print_performance_metrics(self, batch_size):
                 writer = csv.writer(f)
                 writer.writerow(self.__start_times[2:])
```

Comment:
Re "No need to dismiss warm up runs. They are accounted for by print_performance_metrics() func" - you can also see it here

Reply:
Might make sense to introduce num of warm up runs as some global variable being shared by all frameworks' infrastructure as opposed to having it hard-coded in various places.